pkg/zfs/cmd/v1alpha1/zpool/dump/utils_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2019 The OpenEBS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pstatus
+
+import (
+	"testing"
+)
+
+func TestGetDiskStripPath(t *testing.T) {
+	tests := map[string]struct {
+		devPath, stripPath string
+	}{
+		"Test 1": {
+			devPath:   "/dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0-part23",
+			stripPath: "/dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0",
+		},
+		"Test 2": {
+			devPath:   "/dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0p12k",
```

Comment:
test for:
` 			devPath:   "/dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0p21k", `, and,
` 			devPath:   "/dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0p21", `

Reply:
for second one, stripPath need to be `dev/disk/by-id/scsi-0Google_PersistentDisk_persistent-disk-0`

Reply:
ok